lib/positioning/advisory_lock.rb

```diff
@@ -12,17 +12,17 @@ def initialize(base_class, column)
       @column = column.to_s
 
       @adapters = {
-        "Mysql2" => Adapter.new(
+        "mysql2" => Adapter.new(
```

Comment:
I am not sure about this key! Someone should double-check it.

Reply:
https://github.com/rails/rails/blob/b9ca94caea2ca6a6cc09abaffaad67b447134079/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb#L35C7-L35C37

correct